### PR TITLE
Remove vestiges of WScript support

### LINKED ISSUE #14414: Remove vestigates of WScript support
Chatting with @mhegazy, it sounds like WScript is unused at this point.

(Will submit a PR to clean up.)

### PR DESCRIPTION
**Pull request by DLehenbauer**
WSH is no longer in use - Fixes #14414

### CODE REVIEW

**Comment by DLehenbauer** [comment]
Force pushed an update to fix the CRLF mismatch in 'Jakefile.js' that snuck in when I resolved an earlier conflict in this file through Github's web UI.  (The resulting diff is now much smaller.)

File: src/harness/harness.ts [hunk 1]
```diff
@@ -840,16 +752,16 @@ namespace Harness {
...
+    const environment = Utils.getExecutionEnvironment();
+    switch (environment) {
         case Utils.ExecutionEnvironment.Node:
             IO = IOImpl.Node;
             break;
         case Utils.ExecutionEnvironment.Browser:
             IO = IOImpl.Network;
             break;
+        default:
+            throw new Error(`Unknown value '${environment}' for ExecutionEnvironment'.`);
```
Comment (RyanCavanaugh) [inline src/harness/harness.ts]: Infinitesimal nit which should be ignored: Missing (or extraneous) `'` here

Comment (DLehenbauer) [inline src/harness/harness.ts]: Oops.  I'll bundle the fix w/a future PR.  (Thx.)

**Comment by RyanCavanaugh** [comment]
@mhegazy good to merge?

**Pull request merged**